predicators/ml_models.py

```diff
@@ -1375,9 +1380,12 @@ def get_option(self,
                    state: State,
                    goal: Set[GroundAtom],
                    num_samples_per_ground_nsrt: int,
-                   epsilon: float = 0.0) -> _Option:
+                   train_or_test: str = "test") -> _Option:
```

Comment:
so ideally, we wouldn't delete this functionality of being able to directly specify an `epsilon` value. Could we support both by instead introducing some class variable called `use_epsilon_annealing` which will be a boolean set by command line args. Then here, we can check whether `use_epsilon_annealing` is True, and if so, we can use your logic, otherwise we can take the epsilon value that's being passed in?

Reply:
Turns out when the epsilon parameter in get_option here is kept, we would have to have default train_or_test = "test" in rl_bridge's solve function. The tricky part is we want eps annealing to be True for rl_bridge, but then we'll override epsilon to be whatever the annealing epsilon is (which is not 0), since setting train_or_test = "" in Maple Q will overwrite the train_or_test: str = "test" here. We should able to specify epsilon in the command line argument though so hopefully that's ok?

Reply:
Yup, that should be okay! I think if an epsilon is set but a train or test value is also set, we should throw an error or something here just to make sure there isn't an unexpected silent failure